Controller/comments.controller.js

```diff
@@ -7,11 +7,21 @@ exports.getArticleComments = (req, res, next) => {
     const promises = [selectArticle(req.params.article_id), selectArticleComments(req.params.article_id)]
     Promise.all(promises)
     .then((comments) => {
-        console.log(comments);
-        if(comments[1].length === 0){
-           res.status(200).send('No comments yet. Be the first to comment!')
+        if(Array.isArray(comments[1]) && !comments[1].length){
+           res.status(200).send(comments[1])
         }
         res.status(200).send(comments[1])
     }).catch(next)
 }
 
+
+exports.addNewComment = (req, res, next) => {
+    const newComment = req.body
+    const articleID = req.params.article_id
+    const promises = [selectArticle(req.params.article_id), insertComment(newComment, articleID)]
+    Promise.all(promises)
+    .then((comment) => {
+        res.status(201).send(comment[1])
```

Comment:
The comment object should be sent on an appropriate key eg `comment` not just as the body. You will have to adjust your tests first to accomodate this but it is not good practise to do as you are here 👍